### PR TITLE
Adding tacco package

### PR DESCRIPTION
**Pull request by JWatter**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/tacco/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  host:
+    - python >=3.7
+    - setuptools <=60
+    - wheel
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >=3.7
+    - requests
+    - importlib  # [py==26]
```
Comment (dopplershift) [inline recipes/tacco/meta.yaml:27]: I'd just remove this one to keep it noarch. conda-forge isn't building on Python <=3.6 by default anyways.

Comment (JWatter) [reply recipes/tacco/meta.yaml:27]: good point, thanks! I adjusted it accordingly.

Comment (JWatter) [reply recipes/tacco/meta.yaml:27]: I also removed the pip check as it claimed that mkl is not available while it is listed in all requirements and also in the installation schedule

Comment (dopplershift) [reply recipes/tacco/meta.yaml:27]: I don't think MKL is something that pip is capable of detecting.

**Comment by JWatter** [comment]
@conda-forge-admin, please ping conda-forge/help-python
The recipe has passed all checks and is ready for your review. Thanks.

File: recipes/tacco/meta.yaml (hunk 1 above)
Comment (JWatter) [reply recipes/tacco/meta.yaml]: Well, finally got it.. Thanks for the help!

**Review by dopplershift — APPROVED** [review]
Just need confirmation from @simonwm that they are willing to be listed as a maintainer for this recipe.

**Comment by simonwm** [comment]
By all means! It is our first package, sorry for the delay...

**Pull request merged**